src/main/kotlin/me/serce/solidity/ide/run/SearchUtils.kt

```diff
@@ -0,0 +1,20 @@
+package me.serce.solidity.ide.run
+
+import com.intellij.openapi.project.Project
+import com.intellij.psi.stubs.StubIndex
+import me.serce.solidity.lang.psi.SolContractDefinition
+import me.serce.solidity.lang.psi.SolFunctionDefinition
+import me.serce.solidity.lang.psi.SolNamedElement
+import me.serce.solidity.lang.stubs.SolGotoClassIndex
+
+
+object SearchUtils {
+  fun findContract(contractName: String?, project: Project): SolContractDefinition? {
+    val elements = StubIndex.getElements(SolGotoClassIndex.KEY, contractName!!, project, null, SolNamedElement::class.java)
+    val find = elements.find { it is SolContractDefinition } ?: return null
+    return find as SolContractDefinition?
```

Comment:
A small one-liner `return elements.firstOrNull { it is SolContractDefinition } as? SolContractDefinition` might be more readable here

Reply:
Done